### ISSUE TITLE
Not updates on resize

### ISSUE THREAD

**Issue by fareloz** [MurhafSousli/ngx-scrollbar#123]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### Versions
ng-scrollbar 4.1.1

### Repro steps
1. Open official demo on https://murhafsousli.github.io/ngx-scrollbar/
2. Resize browser height
3. Click on scrollbar on the left side
Observed:
2. If there was no scrollbar (all the data is fit in current height without scrolling) it won't appear on resize
3. Scrollbar updates only on click

### Desired functionality
The scrollbar should update it's scrolling capacity on browser resize

**Comment by fperera123** [comment]
![image](https://user-images.githubusercontent.com/31954660/53696079-76201880-3de9-11e9-93d2-4aeeac23d586.png)

Here's too, scroll bars replaced by default ones when width is less than 901

**Comment by MurhafSousli** [comment]
This is fixed in v4.2.0
